app/views/_webhook-trigger-cause.html

```diff
@@ -0,0 +1,9 @@
+{{trigger.message === 'GitHub WebHook' ? 'GitHub webhook' : 'Generic webhook'}} webhook: {{build.spec.revision.git.message}} <osc-git-link
+  class="hash"
+  uri="build.spec.source.git.uri"
+  commit="build.spec.revision.git.commit">{{build.spec.revision.git.commit | limitTo:7}}
+</osc-git-link>
+authored by {{build.spec.revision.git.author.name}}, <a href="" ng-if="!showSecret" ng-click="toggleSecret()">Show obfuscated secret</a>
```

Comment:
i would lowercase the S, it looks weird to me to have a ',' followed by a capital letter